src/main/java/appeng/client/render/effects/LightningFX.java

```diff
@@ -35,6 +35,7 @@
 
 	private static final Random RANDOM_GENERATOR = new Random();
 	private static final int STEPS = 5;
+	private static final int BRIGHTNESS = 13 << 4;
```

Comment:
You can keep it then.
4 bit indendation is to avoid transformations when getting value from lightmap. Probably :D.